appveyor.yml

```diff
@@ -54,16 +54,23 @@ image: Visual Studio 2017
 configuration: Release
 
 cache: 
-#- c:\tools\vcpkg\installed\
-- c:\tools\vcpkg\installed\ -> appveyor.yml
+- c:\tools\vcpkg\installed\
+#- c:\tools\vcpkg\installed\ -> appveyor.yml
 - packages -> **\packages.config
 
 install:
+# update vcpkg
+- cd C:\Tools\vcpkg
+- git pull
+- .\bootstrap-vcpkg.bat
+- cd %APPVEYOR_BUILD_FOLDER%
 # build tesseract
 - cd c:\tools\vcpkg
 - vcpkg integrate install
+- vcpkg remove tesseract:x64-windows-static
```

Comment:
`vcpkg remove --outdated` should remove tesseract if and only if it's out of date, which should save you CI time.

Reply:
Thanks :+1: 

I am in trouble because the old tesseract vcpkg files cached in appveyor is not updated unless the test succeeds. Building both x64 and x86 tesseract reaches appveyor's time limit (60 minutes).